src/otx/core/model/module/base.py

```diff
@@ -205,3 +209,7 @@ def export(self, output_dir: Path, export_format: OTXExportFormat) -> None:
             export_format: Format in which this `OTXModel` is exported.
         """
         self.model.export(output_dir, export_format)
+
+    def forward(self, *args, **kwargs) -> OTXBatchPredEntity | OTXBatchLossEntity:
+        """Model forward pass."""
+        return self.model.forward(*args, **kwargs)
```

Comment:
Why do we need to redefine `forward()` here?

Reply:
`LightningModule` requires forward to be defined for `trainer.predict`.